src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/DefaultRazorDocumentMappingServiceTest.cs

```diff
@@ -32,7 +33,7 @@ public DefaultRazorDocumentMappingServiceTest()
         public void TryMapFromProjectedDocumentRange_Strict_StartOnlyMaps_ReturnsFalse()
         {
             // Arrange
-            var service = new DefaultRazorDocumentMappingService(LoggerFactory);
+            var service = new DefaultRazorDocumentMappingService(TestLanguageServerFeatureOptions.Instance, new TestDocumentContextFactory(), LoggerFactory);
```

Comment:
Ya when I did this for completion I made HEAVY use of the "Change Signature" light bulb 😄. Not sure if you did the same but it's a life saver

Reply:
Yeah, this is all either intellicode, or paste :)